tests/test_ujson.py

```diff
@@ -1026,6 +1028,19 @@ def __str__(self):
         ujson.dumps({Obj(): 1}, sort_keys=sort_keys)
 
 
+def no_memory_leak(func_code, n=None):
+    code = f"import functools, ujson; func = {func_code}"
+    path = os.path.join(os.path.dirname(__file__), "memory.py")
+    n = [str(n)] if n is not None else []
+    p = subprocess.run(["python3", path, code] + n)
```

Comment:
```suggestion
    p = subprocess.run([sys.executable, path, code] + n)
```
There's no guarantee that this current Python environment is in `PATH` or that it's the first one. This will be the cause of `psutil` not getting found on Windows - it's running in the wrong environment.

Reply:
Ah right, good point, thanks!